deploy-as-code/helm/charts/core-services/mdms-v2/values.yaml

```diff
@@ -31,20 +31,18 @@ java-args: "-Dspring.profiles.active=monitoring"
 egov-mdms-schema-definition-save-topic: "save-mdms-schema-definition"
 egov-mdms-data-save-topic: "save-mdms-data"
 # Resource configuration for Go service
-# resources: |
-#   requests:
-#     cpu: "200m"
-#     memory: "512Mi"
-#   limits:
-#     cpu: "200m"
-#     memory: "512Mi"
+resources: |
+  requests:
+    cpu: "200m"
+    memory: "512Mi"
+  limits:
+    cpu: "200m"
+    memory: "512Mi"
 
 # Additional Container Envs
 env: |
   - name: SERVER_CONTEXT_PATH
-    value: {{ .Values.contextPath | default "/mdms-v2" }} 
```

Comment:
@shilpa-egov  why this is getting reverted ? 
cc @SriPadma8997-egov  @nikhilmulinti